e2e/tests/macro-options.spec.ts

```diff
@@ -16,11 +16,12 @@ const test = base.extend({
   },
 })
 
+// The JavaScript is disabled in some of these tests as the iframe
+// resizer causes instability when taking mobile screenshots
 test.describe('macro options', () => {
-  // The iframe resizer is causing instability when taking mobile screenshots
-  test.use({ javaScriptEnabled: false })
-
```

Comment:
This was causing problems when sharding was used. Not sure why but probably relating to one of the nested describe blocks having `javaScriptEnabled: true`.